app/controllers/user_confirmations_controller.rb

```diff
@@ -0,0 +1,37 @@
+class UserConfirmationsController < DeviseController
+  include Spree::Core::ControllerHelpers::Auth # Needed for access to current_ability, so we can authorize! actions
+
+  # GET /resource/confirmation/new
+  def new
+    build_resource({})
+  end
+
+  # POST /resource/confirmation
```

Comment:
No credit here, it's from the [EnterpriseConfirmationsController](https://github.com/openfoodfoundation/openfoodnetwork/blob/master/app/controllers/enterprise_confirmations_controller.rb) that I built upon ^.^